### PR TITLE
feat(site-header): port EU version to EC - INNO-1237

### PR DESCRIPTION
**Pull request by degliwe**
# PR description

Please drop a few lines about the PR: what it does, how to test it, etc.

## QA Checklist

In order to ensure a safe and quick review, please check that your PR follow those guidelines:

* [ ] `package.json` is up-to-date and `@ecl/[system]-base` is part of the dependencies
* [ ] I have checked the dependencies
* [ ] I have given the fractal status “ready” to my component
* [ ] I have declared `@define mycomponent` in the SCSS file
* [ ] I have specified `margin: 0;` on the CSS component
* [ ] I have provided tests
* [ ] I follow the naming guidelines
* [ ] the component supports composition
* [ ] there are no hardcoded strings (all content come from the context)
* [ ] I have filled the README.md file (at least a few lines)
* [ ] My component is listed in the root README
* [ ] My PR has the right label(s)

### CODE REVIEW

File: src/systems/ec/implementations/react/packages/site-header/SiteHeader.jsx [hunk 1]
```diff
@@ -0,0 +1,93 @@
+/* eslint-disable import/no-extraneous-dependencies */
```
Comment (yhuard) [inline src/systems/ec/implementations/react/packages/site-header/SiteHeader.jsx]: Please remove this comment

File: src/systems/ec/implementations/react/packages/site-header/stories/Index.jsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+import demoContent from '@ecl/ec-specs-site-header/demo/data';
+
+import SiteHeader from '../SiteHeader';
+
+storiesOf('SiteHeader', module)
+  .addDecorator(withKnobs)
+  .add('default', () => (
+    <SiteHeader
+      logo={demoContent.logo}
```
Comment (yhuard) [inline src/systems/ec/implementations/react/packages/site-header/stories/Index.jsx]: simpler to use `{...demoContent}`

File: src/systems/ec/implementations/vanilla/packages/ec-component-site-header/README.md [hunk 3]
```diff
@@ -1,16 +1,18 @@
-# Site header
+# EC Site header
 
 ## Why and how to use this component
 
-The site header conveys the EC brand to the users and provides basic structure and guidance. It informs users about which site they are on and their current location in the site. It also provides users with context on the page they're viewing and the site search and language switcher options.
+The site header conveys the EU brand to the users and provides basic structure and guidance. It informs users about which site they are on and their current location in the site. It also provides users with context on the page they're viewing and the site search and language switcher options.
```
Comment (yhuard) [inline src/systems/ec/implementations/vanilla/packages/ec-component-site-header/README.md]: `EC`, not `EU` (unless I've missed something)

File: src/systems/ec/implementations/vanilla/packages/ec-component-site-header/package.json [hunk 4]
```diff
@@ -3,18 +3,13 @@
   "author": "European Commission",
   "license": "EUPL-1.1",
   "version": "2.0.0-alpha.0",
-  "description": "ECL Site headers",
+  "description": "EU Site headers",
```
Comment (yhuard) [inline src/systems/ec/implementations/vanilla/packages/ec-component-site-header/package.json]: `EC`, not `EU`

**Review by yhuard — CHANGES_REQUESTED** [review]

File: src/systems/ec/implementations/react/packages/site-header/SiteHeader.jsx [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+  const classNames = classnames(className, {
+    'ecl-site-header': true,
+  });
+
+  return (
+    <header {...props} className={classNames}>
+      <div className="ecl-site-header__container ecl-container">
+        <div className="ecl-site-header__banner">
+          <a
+            className="ecl-site-header__logo ecl-link ecl-link--standalone"
```
Comment (yhuard) [inline src/systems/ec/implementations/react/packages/site-header/SiteHeader.jsx]: Would it have been possible to use the Link component here?

File: src/systems/ec/implementations/react/packages/site-header/SiteHeader.jsx [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+            href={logo.href}
+            aria-label={logo.title}
+          >
+            <Logo {...logo} />
+          </a>
+
+          <div className="ecl-site-header__selector">
+            <a
+              className="ecl-link ecl-link--standalone"
```
Comment (yhuard) [inline src/systems/ec/implementations/react/packages/site-header/SiteHeader.jsx]: Would it have been possible to use the Link component here?

File: src/systems/ec/implementations/vanilla/packages/ec-component-site-header/README.md [hunk 7]
```diff
@@ -1,16 +1,18 @@
...
 
 ## Why and how to use this component
 
 The site header conveys the EC brand to the users and provides basic structure and guidance. It informs users about which site they are on and their current location in the site. It also provides users with context on the page they're viewing and the site search and language switcher options.
 
 ## When to use this component
 
-* on every page of every Commission website
-* detail and landing pages
+on every page of Europa sites
```
Comment (yhuard) [inline src/systems/ec/implementations/vanilla/packages/ec-component-site-header/README.md]: The docs are for the EU version. Please fix 'em :)

File: src/systems/ec/specs/components/site-header/docs/code.mdx [hunk 8]
```diff
@@ -0,0 +1,12 @@
+import Playground from '@ecl/website-components/Playground'
+import SiteHeader from '@ecl/ec-react-component-site-header/SiteHeader'
+
+import demoContent from '../demo/data';
+
+<Playground playgroundLink="/storybook/ec/index.html?selectedKind=SiteHeader&selectedStory=default&stories=0">
+  <SiteHeader
+    logo={demoContent.logo}
+    languageSelector={demoContent.languageSelector}
```
Comment (yhuard) [inline src/systems/ec/specs/components/site-header/docs/code.mdx]: Please `{...demoContent}`

File: src/systems/ec/implementations/vanilla/packages/ec-component-site-header/package.json [hunk 9]
```diff
@@ -3,18 +3,13 @@
   "author": "European Commission",
   "license": "EUPL-1.1",
   "version": "2.0.0-alpha.0",
-  "description": "ECL Site headers",
+  "description": "EC Site headers",
```
Comment (yhuard) [inline src/systems/ec/implementations/vanilla/packages/ec-component-site-header/package.json]: `"EC Site headers"` -> `"ECL EC Site headers"` (for consistency)

**Review by yhuard — CHANGES_REQUESTED** [review]

File: src/systems/ec/implementations/react/packages/site-header/SiteHeader.jsx (hunk 5 above)
Comment (degliwe) [reply src/systems/ec/implementations/react/packages/site-header/SiteHeader.jsx]: link component not suitable for this

File: src/systems/ec/implementations/react/packages/site-header/SiteHeader.jsx (hunk 6 above)
Comment (degliwe) [reply src/systems/ec/implementations/react/packages/site-header/SiteHeader.jsx]: link component not suitable for this

**Review by yhuard — APPROVED** [review]

**Pull request merged**